utmap-server/README.html

```diff
@@ -197,25 +197,25 @@ <h3>Table of Contents </h3>
   <h4><a href="#Events">Events</a></h4>
   <ul>
   <li><a href="#addEvent"><code><span class="http-method">post</span> /event</code></a></li>
-  <li><a href="#deleteEvent"><code><span class="http-method">delete</span> /event/{eventName}</code></a></li>
```

Comment:
Minor note if these are python functions, that python uses snake_case. I'm not sure how significant styling is though, you could bring it up with Shahmeer.

Reply:
Can do.

Reply:
I'll give my approval for the pull request once this is resolved (After he gets back to you)